extensions/wikia/AdEngine/js/video/vastUrlBuilder.js

```diff
@@ -1,5 +1,6 @@
 /*global define*/
 define('ext.wikia.adEngine.video.vastUrlBuilder', [
+	'ext.wikia.adEngine.adContext',
```

Comment:
What are the reasons of having this dependency if we don't use it? :japanese_ogre: